src/main/java/com/work/management/web/rest/controller/TeamController.java

```diff
@@ -0,0 +1,34 @@
+package com.work.management.web.rest.controller;
+
+import com.work.management.web.rest.resource.TeamResource;
+import com.work.management.service.impl.TeamService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "workmanagement/v1/teams")
+public final class TeamController {
+
+
```

Comment:
Remove empty lines from here... there should just be one empty line between the line 18 and 21.